lightbulb/app.py

```diff
@@ -947,7 +949,8 @@ async def get_prefix_context(
             ctx._parser = (ctx.command.parser or parser.Parser)(ctx, args)
         return ctx
 
-    async def process_prefix_commands(self, context: context_.prefix.PrefixContext) -> None:
+    @staticmethod
+    async def process_prefix_commands(context: context_.prefix.PrefixContext) -> None:
```

Comment:
Out of scope for this pr. This was left as an instance method in case people wish to override it.

Reply:
I do not particularly like this change either, as I plan on overriding such methods for customizing command processing myself.

Reply:
> Out of scope for this pr. This was left as an instance method in case people wish to override it.

`staticmethods` can still be overwritten. They just, in this case, dont take in a self parameter, making them a tad faster. But will make another pr for this